app/models/concerns/facets/hostgroup_extensions.rb

```diff
@@ -7,6 +7,7 @@ module HostgroupExtensions
 
     included do
       configure_facet(:hostgroup, :hostgroup, :hostgroup_id)
+      refresh_facet_relations
```

Comment:
It would seem so, but for some reason it's not true.
We are doing the same here for `Host`: https://github.com/theforeman/foreman/blob/c5e0048a782643814f0f95ce8f08b993f9fb9465/app/models/concerns/facets/base_host_extensions.rb#L9
Could it be caused by model not including it directly, so the included block of `ModelExtensionBase` is actually executed on the module not the model?
Should we move the `include Facets::ModelExtenstionsBase` statement to the `included` block for both `Host` and `Hostgroup`?

Reply:
> Should we move the include Facets::ModelExtenstionsBase statement to the included block for both Host and Hostgroup?

Makes sense. Can you try it? If it works - it's a win, if it doesn't let's leave the `refresh_facet_relations` in hostgroup too.